### ISSUE TITLE
Language translation

### ISSUE THREAD

**Issue by lamuye** [statamic/cms#9343]
### Bug description

The widgets title is not getting translated based on the json file

### How to reproduce

1. Create a en.json file in resources/lang directory
2. in the json file, you can write 

{
  "Pages" : "sider"
}

### Environment

```yaml
APP_NAME=Properti
APP_ENV=production
APP_KEY=
APP_DEBUG=false
APP_URL=http://localhost
APP_LOCALE=
APP_TIMEZONE=Europe/Oslo
PROPERTI_LOCALE=

STATAMIC_LICENSE_KEY=
STATAMIC_STACHE_WATCHER=true
STATAMIC_STATIC_CACHING_STRATEGY=null
STATAMIC_REVISIONS_ENABLED=false
STATAMIC_GRAPHQL_ENABLED=false
STATAMIC_API_ENABLED=false
STATAMIC_GIT_ENABLED=false

LOG_CHANNEL=stack
LOG_DEPRECATIONS_CHANNEL=null
LOG_LEVEL=debug

DB_CONNECTION=mysql
DB_HOST=127.0.0.1
DB_PORT=3306
DB_DATABASE=laravel
DB_USERNAME=root
DB_PASSWORD=

BROADCAST_DRIVER=log
CACHE_DRIVER=file
FILESYSTEM_DISK=local
QUEUE_CONNECTION=sync
SESSION_DRIVER=file
SESSION_LIFETIME=120

MEMCACHED_HOST=127.0.0.1

REDIS_HOST=127.0.0.1
REDIS_PASSWORD=null
REDIS_PORT=6379

MAIL_MAILER=smtp
MAIL_HOST=mailpit
MAIL_PORT=1025
MAIL_USERNAME=null
MAIL_PASSWORD=null
MAIL_ENCRYPTION=null
MAIL_FROM_ADDRESS="hello@example.com"
MAIL_FROM_NAME="${APP_NAME}"

AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1
AWS_BUCKET=
AWS_USE_PATH_STYLE_ENDPOINT=false

PUSHER_APP_ID=
PUSHER_APP_KEY=
PUSHER_APP_SECRET=
PUSHER_HOST=
PUSHER_PORT=443
PUSHER_SCHEME=https
PUSHER_APP_CLUSTER=mt1

VITE_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
VITE_PUSHER_HOST="${PUSHER_HOST}"
VITE_PUSHER_PORT="${PUSHER_PORT}"
VITE_PUSHER_SCHEME="${PUSHER_SCHEME}"
VITE_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"

DEBUGBAR_ENABLED=false
```

### Installation

Fresh statamic/statamic site via CLI

### Antlers Parser

Runtime (default)

**Comment by jasonvarga** [comment]
Which widget?

**Comment by lamuye** [comment]
> Which widget?

collections

**Comment by jasonvarga** [comment]
It seems to work fine. What version of Statamic are you using?

Also, when you create an issue, in the "Environment" section we want details about your setup (the output of the `php please support:details` command), not your `.env` file.

**Comment by lamuye** [comment]
Environment
Application Name: Properti Stage
Laravel Version: 9.52.15
PHP Version: 8.1.15
Composer Version: 2.6-dev+29134d5a1d268098fb254bbed9081a335e7071f3
Environment: local
Debug Mode: ENABLED
URL: properti-stage.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 7
Antlers: runtime
Stache Watcher: Disabled
Static Caching: Disabled
Version: 4.36.0 PRO

Statamic Addons
aryehraber/statamic-captcha: 1.10.0
properti/estate-items-import: dev-main
properti/integrations: dev-main
properti/properti-dashboard: dev-main
r8edge/imagemap: dev-main
r8edge/welcome-widget: dev-main
statamic/eloquent-driver: 2.6.1

Statamic Eloquent Driver
Asset Containers: file
Assets: eloquent
Blueprints: eloquent
Collection Trees: file
Collections: eloquent
Entries: eloquent
Forms: eloquent
Global Sets: eloquent
Global Variables: file
Navigation Trees: file
Navigations: eloquent
Revisions: eloquent
Taxonomies: eloquent
Terms: eloquent

**Comment by jasonvarga** [comment]
The widget translation issue was fixed in 4.37.0. https://github.com/statamic/cms/pull/9077

Please upgrade and your issue will be fixed.

In the future please try upgrading before reporting bugs. They might already be fixed.
